assets/systemd-unit-files/monero-wallet-rpc.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Monero Wallet RPC
+After=network.target monerod.service
+
+[Service]
+User=moneroservices
+Group=moneroservices
+WorkingDirectory=/opt/monero-wallets
+Type=simple
+ExecStart=/opt/monero-bin/monero-wallet-rpc --wallet-file /opt/monero-wallets/woocommerce --rpc-bind-port 18080 --password-file /opt/monero-wallets/woocommerce.password --disable-rpc-login --log-file /var/log/monero-wallet.log
```

Comment:
--disable-rpc-login ? Do you know what will happen if the rpc-login is disabled?

Reply:
Am I missing where the original WP plugin used authorization? Not being snarky.